src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -27,21 +27,21 @@ public TorpedoStore(int numberOfTorpedos){
       }
     }
   }
-
+  Random generator = new Random();
```

Comment:
This is correct, the variable has been made global, to avoid creating it each time